react/Pill/Pill.less

```diff
@@ -0,0 +1,87 @@
+@import (reference) '~seek-style-guide/theme';
+
+@staticTagHeight: @row-height * 5;
+@interactiveTagHeight: @row-height * 8;
+@removeCircleDiameter: @row-height * 4;
+
+.staticPill {
+  .rawText(@interaction-type-scale);
+
+  align-items: center;
+  display: inline-flex;
+  color: @sk-charcoal;
+  background: @sk-gray-light;
+  border-radius: @staticTagHeight / 2;
+  height: @staticTagHeight;
+  line-height: @staticTagHeight;
+  padding: 0 15px;
+  margin: @row-height 10px @row-height 0;
+}
+
+.interactivePill {
+  align-items: center;
+  display: inline-flex;
+  font-size: 18px;
```

Comment:
We shouldn't have explicit font sizes specified like this, happy to pair with you if there is a problem that you're working around

Reply:
Turns out that's not needed at all, it was left over from some CSS that was copy / pasted from profile. `<Text>` is controlling the font size